teams/roles-and-responsibilities.md

```diff
@@ -1 +1,65 @@
-# Teams: Roles and Responsibilities
+# Ember Core Team Member Roles, Responsibilities, and Expectations
+
+## Participation
+
+As a bare minimum, core team members are expected to:
```

Comment:
Is this an "any of these" or "all of these" list? It seems that if we applied this rubric as "all of these" to todays core team members, we would eliminate at least half of them (but probably much more)?

Reply:
it's more of a "some combination of these"

Reply:
Like a "at least 2 of these"? Maybe we could figure different wording we can use to make that a bit clearer? If we could (and we keep the self assessment thing) then having it be clearer would help..

Reply:
It has to be a self-identified number of things. Everyone will have their own perspective on how much they could/would/should take on, and I think the point is that they figure that out for themselves, and we work with what volunteers we have.